packages/ubuntu_bootstrap/integration_test/screenshot_test.dart

```diff
@@ -43,9 +43,7 @@ Future<void> main() async {
       await tester.runApp(() => runInstallerApp([], theme: currentTheme));
       await tester.pumpAndSettle();
 
-      await tester.testLocalePage(
-        screenshot: '$currentThemeName/locale',
-      );
+      await tester.testLocalePage(screenshot: '$currentThemeName/locale');
```

Comment:
1) For some reason, we don't require the navigation flag in certain bootstrap integration tests